src/app.js

```diff
@@ -25,107 +25,92 @@ export function app(props) {
     }
   }
 
-  function render(cb) {
-    element = patch(
-      appRoot,
-      element,
-      node,
-      (node = appView(appState, appActions)),
-      (skipRender = !skipRender)
+  function render() {
+    flush(
+      (element = patch(
+        appRoot,
+        element,
+        node,
+        (node = appView(appState, appActions)),
+        (skipRender = !skipRender)
+      ))
     )
-    while ((cb = lifecycleCallbackStack.pop())) cb()
   }
 
-  function hydrate(element, map) {
-    return (
-      element &&
-      h(
-        element.tagName.toLowerCase(),
-        {},
-        map.call(element.childNodes, function(element) {
-          return element.nodeType === 3
-            ? element.nodeValue
-            : hydrate(element, map)
-        })
-      )
-    )
+  function flush(cb) {
+    while ((cb = callbackStack.pop())) cb()
+  }
+
+  function setup(module, state, actions) {
+    copy(state, module.state)
+
+    setupActions(actions, state, module.actions)
+
+    if (module.init) {
+      callbackStack.push(function() {
```

Comment:
To get some bytes back, maybe use bind here?

Reply:
@pspeter3 Thanks, I'll see if I can squeeze this any further. 🤓

Reply:
`bind` is adding more byte than `function(){ ...` here Oo